sdjournal/journal.go

```diff
@@ -1155,7 +1155,6 @@ func (j *Journal) GetBootID() (string, error) {
 		return "", err
 	}
 
-	id128StringMax := C.ulong(C.SD_ID128_STRING_MAX)
```

Comment:
```
id128StringMax := C.size_t(C.SD_ID128_STRING_MAX)
```

should be enough to fix this everywhere. Can you try a scratch build on 32b just to confirm, and if so amend the patch?

Reply:
that worked.